### ISSUE TITLE
NVDA announcing  Unwanted info after every suggestion in vscode 

### ISSUE THREAD

**Issue by balajip881** [nvaccess/nvda#15565]
### Is your feature request related to a problem? Please describe.

I use VS code for coding. While editing in a file let say test.js
after typing fun, suggestion  function appears
press tab accepts the suggestion 
after every suggestion, NVDA reading out  file name and auto complete and line content.

I raised the same in vscode. but they  told me to raise the issue in NVDA .
After every suggestion, Focus is jumping from quick suggestion widjet to  texteditor, there by announcing the unwanted info.

You can track the vscode issue here.

https://github.com/microsoft/vscode/issues/194255

**Comment by balajip881** [comment]
After returning the focus from quick suggestions widget to  editor, 
NVDA is reading out file name and line content. It is extra info for every suggestion.

###Solution : 
1. I need a setting in NVDA so that NVDA wont announce role state and contents  of a particular text area.
2. Please let us know if there is any ARIA property, so that NVDA wont announce anything after refocusing on to editor text area.

**Comment by seanbudd** [comment]
Welcome @balajip881 

We are unable to process this issue as it stands.

This is a bug report. As such, a lot of important information is missing such as NVDA version and VS code version used to reproduce this issue.
Please fill out a new issue using the bug report template fully.


Kind Regards,
NV Access Software Developers
